Food/terraform-docs.lua

```diff
@@ -12,7 +12,7 @@ food = {
             arch = "amd64",
             url = "https://github.com/segmentio/" .. name .. "/releases/download/v" .. version .. "/" .. name .. "-v" .. version .. "-darwin-amd64",
```

Comment:
- `terraform-docs` has been migrated from `segmentio` to its own `terraform-docs` organization.
- since [v0.11.0](https://github.com/terraform-docs/terraform-docs/releases/tag/v0.11.0) we've started using GoReleaser to facilitate the release chores and as part of that we've moved from distributing standalone binaries to compressed files (`tar.gz` and `zip`). We're going to deprecate standalone binaries in the upcoming releases.